quiz-backend/src/Quiz.cpp

```diff
@@ -1,45 +1,66 @@
-#include "Quiz.h"
-#include "User.h"
+#include "quiz.h"
+#include "user.h"
 #include <utility>
 #include "Question.h"
```

Comment:
nit: if we revert the uppercase names for the quiz and user header files I think some consistency for "Question.h" would be great as well.